static/app/components/modals/inviteMembersModal/inviteStatusMessage.tsx

```diff
@@ -39,12 +43,45 @@ export default function InviteStatusMessage({
     const errorCount = statuses.filter(i => i.error).length;
 
     if (willInvite) {
-      const invites = <strong>{tn('%s invite', '%s invites', sentCount)}</strong>;
+      const invites = (
+        <strong data-test-id="sent-invites">
+          {tn('%s invite', '%s invites', sentCount)}
+        </strong>
+      );
+      const failedInvites = (
+        <strong data-test-id="failed-invites">
+          {tn('%s invite', '%s invites', errorCount)}
+        </strong>
+      );
       const tctComponents = {
         invites,
         failed: errorCount,
+        failedInvites,
       };
 
+      if (isNewInviteModal) {
```

Comment:
a god rule of thumb in react code, is that if you're returning some JSX, that it can likely be broken down into a component. so tihs the status, etc could likely be pulled out.